site_tools/extra/extra.py

```diff
@@ -30,6 +30,19 @@
 # pylint: enable=no-name-in-module
 # pylint: enable=import-error
 
+def supports_custom_format(clang_exe):
+    """Get the version of clang-format"""
+    import subprocess
+    import re
+
+    output = subprocess.check_output([clang_exe, "--version"])
+    match = re.search("version (\d+)\.", output)
```

Comment:
(lint) '\d'. String constant might be missing an r prefix.

(lint) '\.'. String constant might be missing an r prefix.